lib/ansible/cli/doc.py

```diff
@@ -1358,7 +1358,7 @@ def get_role_man_text(self, role, role_json):
                     text.append(doc.pop('examples').strip())
                 else:
                     try:
-                        text.append(yaml_dump(doc.pop('examples'), indent=2, default_flow_style=False))
+                        text.append(DocCLI._dump_yaml(doc.pop('examples')))
```

Comment:
There is no test coverage for this branch. Can you add some tests for it?

Reply:
i thought i did, that was the purpose of the testinv.yml and test_win_module2.yml files

Reply:
This code is part of `get_role_man_text()`, so it needs a role entrypoint to test this.

Reply:
ah, missed that, did the parity side on modules, I'll add a role test

Reply:
actually removed as 'examples' made no sense and was not part of intent of role argument specs